index.js

```diff
@@ -1 +1,49 @@
-console.log("Hello World!")
\ No newline at end of file
+const axios = require('axios');
+
+const NASA_ASTEROIDS_FEED_URL = 'https://api.nasa.gov/neo/rest/v1/feed';
+const API_KEY = 'eLmnePyuHtTVKYHiKBC7fVH4weBUtTMiZIGoZftC';
+
+const getCurrentWeek = () => {
+    const today = new Date();
+    const dayOfWeek = today.getDay();
+    const monday = new Date(today);
+    monday.setDate(today.getDate() - (dayOfWeek === 0 ? 6 : dayOfWeek - 1));
+    monday.setHours(1, 0, 0, 0);
```

Comment:
[date-fns ](https://date-fns.org/) can help when working with dates and the code will be easier to understand

Reply:
thanks! code looks better